### ISSUE TITLE
Dungeon 1: allow to burn the mario painting

### ISSUE THREAD

**Issue by christopho** [solarus-games/zelda-xd2-mercuris-chess#9]

**Comment by Diarandor** [comment]
I don't remember about this but, if we tap the painting with the sword, does the sound appear like in walls destructible by bombs? I will have to check if this happens or not, because it is convenient to add that sound.

**Comment by christopho** [comment]
The sound is not played when tapping with the sword. Which is another bug, it was reported in the old repository.

**Comment by Diarandor** [comment]
Yes, I remember that I had read about this bug, and since it is related to the same thing it deserved to be mentioned here.
